src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/whitespace/genericwhitespace/Example1.java

```diff
@@ -0,0 +1,59 @@
+/*xml
+<module name="Checker">
+  <module name="TreeWalker">
+    <module name="GenericWhitespace"/>
+  </module>
+</module>
+*/
+package com.puppycrawl.tools.checkstyle.checks.whitespace.genericwhitespace;
+
+import com.google.common.collect.ImmutableList;
+import java.util.Collections;
+import java.util.Comparator;
+import java.util.List;
+import java.util.ArrayList;
+import java.util.Map;
+
+// xdoc section -- start
+public class Example1 {
+  @FunctionalInterface
+  interface ImmListBldr<T> {
+      ImmutableList.Builder<T> createBuilder();
+  }
+  // Generic methods definitions
+  public static <K, V> boolean foo(K key, V value) {
```

Comment:
```suggestion
  public static <K, V> boolean isFoo(K key, V value) {
```

Please do for all similar methods, methods with boolean return values should nearly always be `is...` or `has...`

Reply:
solved